webapp/src/shared/react/useListQueryControls.js

```diff
@@ -3,7 +3,7 @@ import { useSearchParams } from "react-router-dom";
 
 export default function useListQueryControls() {
   const [params, setParams] = useSearchParams(new URLSearchParams());
-  const page = Number(params.get("page") || "0");
+  const page = Number(params.get("page") || "1");
```

Comment:
I think pages are 0-based? What's up with this change? Also it's in shared so you need to copy this to admin and test it there.